<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smartcalc</class>
 <widget class="QWidget" name="smartcalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>781</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>531</width>
    <height>781</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>531</width>
    <height>781</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="right_bracket">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cos">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>580</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_min">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>640</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(0, 0, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_asin">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>520</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>asin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ln">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>700</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>ln</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_C">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>460</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255, 69, 0);
    color:  rgb(255, 255, 255);
    font-size: 25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="left_bracket">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>461</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_percent">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>640</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sqr">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>700</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>^</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sin">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>520</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_back">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>520</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255, 69, 0);
    color:  rgb(255, 255, 255);
    font-size: 25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>640</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(0, 0, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>520</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(0, 0, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="point">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>700</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(0, 0, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sqrt">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>580</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>sqrt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_atan">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>640</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>atan</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{ background: rgb(245, 245, 245);
selection-background-color: rgb(245, 245, 245);
border: 1px solid #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>580</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(0, 0, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(0, 0, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tan">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>640</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>tan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_div">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>520</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_plu">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>700</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>700</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(0, 0, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(0, 0, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mul">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>580</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>640</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(0, 0, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equal_sign">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>700</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(0, 100, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_acos">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>580</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>acos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>580</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(0, 0, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(0, 0, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_log">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>700</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copy_digit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>62</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255, 69, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="graph_widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>431</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>521</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_Xmin">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Xmin =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Xmin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>-10</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Xmax">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Xmax =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Xmax">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Ymin">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Ymin =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Ymin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>-10</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Ymax">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Ymax =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Ymax">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Create_Plot_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255, 69, 0);
    color:  rgb(255, 255, 255);
    font-size:  25px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>Create Plot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Xmax_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>460</y>
     <width>60</width>
     <height>51</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;X =&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{ background: rgb(245, 245, 245);
selection-background-color: rgb(245, 245, 245);
border: 1px solid #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
